source/docs/magento2/advanced.md

```diff
@@ -14,3 +14,20 @@ _Since version 2.7.0_
 Front-Commerce allows you to send additional headers in all API calls. To do so, you must define the `magento.api.extraHeaders` (for storefront API) and/or `magento.api.extraAdminHeaders` (for admin API) configuration values from a [configuration provider](/docs/advanced/server/configurations.html#What-is-a-configuration-provider).
 
 These additional headers could be useful if you want to add additional context to your queries, depending on the request or to detect Front-Commerce requests from your Magento server.
+
+## Clearing Front-Commerce cache
+
+There are two ways to clear the magento cache:
+
+1. Run the below command from the command line on the magento server:
+
+```sh
+bin/magento cache:clean -- front-commerce
+```
+
+2. From the admin interface go to System > Cache Management then select Front-commerce make sure that refresh is selected and then click The Submit button. (see below screenshot for more details).
```

Comment:
![image](https://user-images.githubusercontent.com/305563/151804042-1222b964-95e3-4c34-abfb-a08942784b57.png)

it's rendered as two `1.` items.

Also
```suggestion
2. From the admin interface go to System > Cache Management then select Front-Commerce, make sure that refresh is selected and then click the Submit button. (see below screenshot for more details).
```